procedures/release-duty/index.rst

```diff
@@ -64,7 +64,6 @@ Email
 As ReleaseDuty you need to *subscribe* to certain mailing lists:
 
 *  All types of sign-offs and approvals should go to `release signoff mailing list <https://mail.mozilla.org/listinfo/release-signoff>`__
-*  All releng automation emails should go to `release-automation-notifications <https://groups.google.com/a/mozilla.com/forum/?hl=en#!forum/release-automation-notifications>`__
```

Comment:
We tend to send them to `release+___@m.c` emails now, e.g. https://wiki.mozilla.org/ReleaseEngineering/How_To/Process_release_email (this page has a bunch of cruft, but some valid emails are here as well). We didn't add this to the list of mailing lists to subscribe to, since `release@m.c` was the default releng email list pre-layoffs. We created the `releng@m.c` email list to avoid spamming all the new team members with release automation email, but we may add you all to `release@m.c` as we cross-train on releaseduty.